region/maridia/inner-pink.json

```diff
@@ -1303,17 +1377,33 @@
                       "can2HighWallMidAirMorph",
                       "canSuitlessMaridia"
                     ]}
-                  ]},
-                  {"and": [
+                  ],
+                  "note": [
+                    "All those options are because Takking off Gravity to mid-air morph has been locked behind canSuitlessMaridia.",
+                    "Is that reasonable?"
+                  ]
+                },
+                {
+                  "name": "Suitless Jump Assist",
+                  "notable": false,
+                  "requires": [
                     "canSuitlessMaridia",
                     "Morph",
                     {"or": [
                       "canSpringBallJumpMidAir",
-                      "HiJump",
-                      "canUseFrozenEnemies"
+                      "HiJump"
                     ]}
-                  ]}
-                ]}
+                  ]
+                },
+                {
+                  "name": "Suitless Frozen Zoa",
```

Comment:
This is either a mistake, and there should be a junction between 5 and 4 that this is trying to reach. 
Or this is a cosmic brain freeze the bug 1 tile up inside the maze (charge buffer holding down and release charge when hit), downgrab to land on it, morph and wait for it to unfreeze to dboost up the 2nd tile, repeat, then drop onto the bug to cross the final gap to the item.

Reply:
Forgot jump+morph was a thing, leaving the comment because it was a fun find.

Reply:
That is one crazy strat. I love SM for all the different tools you have for problem solving. I think that's part of what makes entrando so appealing, new strats to come up with.